agenta-cli/agenta/docker/docker-assets/Dockerfile.template

```diff
@@ -7,12 +7,9 @@ WORKDIR /app
 
 COPY . .
 
-RUN pip install --upgrade pip && \
-    pip install --no-cache-dir -r requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
 
-RUN pip install fastapi uvicorn python-dotenv
 EXPOSE 80
 
 CMD ["./entrypoint.sh"]
-# uvicorn agenta_backend.main:app --reload --host 0.0.0.0 --port 8881
-# TODO: add root_path as env variable and give it to the docker to solve the docs issue
\ No newline at end of file
+# uvicorn agenta_backend.main:app --reload --host 0.0.0.0 --port 8881
```

Comment:
This line is not needed. 

 `CMD ["./entrypoint.sh"]` is what we use to start the application.